Router.svelte

```diff
@@ -269,9 +316,10 @@ class RouteItem {
             throw Error('Invalid value for "path" argument')
         }
 
-        const {pattern, keys} = regexparam(path)
+        // At this point path is valid and either a string of RegExp, so we can extract the pattern and path params
+        const {pattern, keys} = parsePath(path)
 
-        this.path = path
+        this.path = path // TODO remove unused
```

Comment:
This is used in the `detail` object for events (see line 501, or 453 before changes)

Reply:
Woops, didn't see that in VSCode, will remove